crates/turbopack-css/src/process.rs

```diff
@@ -878,11 +906,38 @@ impl swc_core::css::visit::Visit for CssValidator {
 }
 
 /// We only vist top-level selectors.
-impl lightningcss::visitor::Visitor<'_> for CssValidator {
+impl lightningcss::visitor::Visitor<'_> for CssModuleValidator {
     type Error = ();
 
     fn visit_types(&self) -> lightningcss::visitor::VisitTypes {
-        visit_types!(SELECTORS)
+        lightningcss::visitor::VisitTypes::all()
+    }
+
+    fn visit_property(
+        &mut self,
+        property: &mut lightningcss::properties::Property,
+    ) -> Result<(), Self::Error> {
+        match property {
+            lightningcss::properties::Property::GridArea(p) => {
```

Comment:
Is this handling these properties too?
[grid-row-start](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-row-start)
[grid-column-start](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-column-start)
[grid-row-end](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-row-end)
[grid-column-end](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-column-end)

Could we have a test case for that?